airflow/providers/apache/spark/operators/spark_sql.py

```diff
@@ -56,6 +58,8 @@ class SparkSqlOperator(BaseOperator):
     :param verbose: Whether to pass the verbose flag to spark-sql
     :type verbose: bool
     :param yarn_queue: The YARN queue to submit to (Default: "default")
+        This parameter has been deprecated. Master and connection parameters (such as YARN queue)
```

Comment:
I don't think we want to deprecate this --  it is a valid option to override the queue for just a single operator for instance.